### PR TITLE
Ms/1095 history user not recorded in history objects

### PR DESCRIPTION
**Pull request by sennierer**

### CODE REVIEW

File: docs/source/features.rst [hunk 1]
```diff
@@ -11,6 +11,25 @@ The :py:mod:`apis_core.history` module provides versioning
 for APIS. It is based on the `django-simple-history`_
 package.
 
+Track changes by users
+^^^^^^^^^^^^^^^^^^^^^^
+
+To record the user who made a change, the `django-simple-history` module 
+used by APIS provides a middleware that sets the `history_user` attribute of 
+the request object to the current user. This attribute is then used by the
```
Comment (b1rger) [inline docs/source/features.rst:19]:
```suggestion
the request object to the current user. It then saves this information to the `history_user` attribute of the historical record. 
```

File: docs/source/features.rst [hunk 2]
```diff
@@ -11,6 +11,25 @@ The :py:mod:`apis_core.history` module provides versioning
 for APIS. It is based on the `django-simple-history`_
 package.
 
+Track changes by users
+^^^^^^^^^^^^^^^^^^^^^^
+
+To record the user who made a change, the `django-simple-history` module 
+used by APIS provides a middleware that sets the `history_user` attribute of 
+the request object to the current user. This attribute is then used by the 
+`VersionMixin` to set the `history_user` attribute of the historical record.
```
Comment (b1rger) [inline docs/source/features.rst:20]:
```suggestion
```

File: docs/source/features.rst [hunk 3]
```diff
@@ -11,6 +11,25 @@ The :py:mod:`apis_core.history` module provides versioning
...
+the request object to the current user. This attribute is then used by the 
+`VersionMixin` to set the `history_user` attribute of the historical record. 
+To activate the middleware, add `simple_history.middleware.HistoryRequestMiddleware` 
+to your `MIDDLEWARE` setting:
+
+.. code-block:: python
+
+    MIDDLEWARE = [
+        ...
+        'apis_core.history.middleware.HistoryMiddleware',
```
Comment (b1rger) [inline docs/source/features.rst:28]:
```suggestion
        'apis_core.history.middleware.HistoryRequestMiddleware',
```

**Review by b1rger — CHANGES_REQUESTED** [review]
please rebase ;)

**Comment by b1rger** [comment]
(I'm not sure if this is also something we want in our default settings - probably depends if we also want the `apis_core.history` app in the `INSTALLED_APPS`)

**Comment by sennierer** [comment]
given that this is recording user names I suggest to just add the comments in the documentation on how to use it

**Comment by sennierer** [comment]
> (I'm not sure if this is also something we want in our default settings - probably depends if we also want the `apis_core.history` app in the `INSTALLED_APPS`)

lets discuss that in another issue

**Comment by sennierer** [comment]
@b1rger rebased it on main and added your suggested changes. I suggest to merge this and discuss whether to add it to default settings or not in another issue

**Review by b1rger — APPROVED** [review]

**Pull request merged**